src/main/models/model-manager.ts

```diff
@@ -480,40 +480,12 @@ export class ModelManager {
     return strategy.getAiderMapping(provider, modelId);
   }
 
-  getModel(providerId: string, modelId: string): Model | undefined {
```

Comment:
I believe, by resolving the conflicts, this function (and some other latest changes) have been removed.